tests/modules/asset_groups/resources/test_create_asset_group.py

```diff
@@ -254,22 +277,52 @@ def test_create_asset_group_detection(
         data.add_filename(0, test_filename)
         data.set_field('speciesDetectionModel', ['someSortOfModel'])
 
-        resp_msg = 'failed to start detection'
-        asset_group_utils.create_asset_group(
-            flask_app_client, None, data.get(), 400, resp_msg
-        )
-        # TODO this is what the response should be
-        # resp_msg = 'Invalid submitter data'
-        # asset_group_utils.create_asset_group(
-        #    flask_app_client, None, data.get(), 403, resp_msg
-        # )
-
+        with mock.patch.object(
+            flask_app.acm,
+            'request_passthrough_result',
+            side_effect=detection_started,
+        ):
+            resp = asset_group_utils.create_asset_group(
+                flask_app_client, None, data.get()
+            )
+            asset_group_uuid = resp.json['guid']
+
+        from invoke import MockContext
+        import io
+
+        with mock.patch('app.create_app'):
+            with mock.patch('sys.stdout', new=io.StringIO()) as stdout:
+                from tasks.app import job_control
+
+                job_control.print_all_asset_jobs(
+                    MockContext(), resp.json['assets'][0]['guid']
+                )
+                job_output = stdout.getvalue()
+                assert 'Job ' in job_output
+                assert 'Active:True Started (UTC)' in job_output
+                assert 'model:someSortOfModel' in job_output
+                with mock.patch.object(
+                    flask_app.acm,
+                    'request_passthrough_result',
+                    side_effect=job_response,
+                ):
+                    job_control.print_last_asset_job(
+                        MockContext(), resp.json['assets'][0]['guid'], verbose=True
+                    )
+                    job_output = stdout.getvalue()
+                    assert 'Job ' in job_output
+                    assert 'Active:True Started (UTC)' in job_output
+                    assert 'model:someSortOfModel' in job_output
+                    # These are on separate lines so are not available in stdout...
```

Comment:
I checked and the asserts actually work so this line can be removed and the next 2 lines can be uncommented

Reply:
Yeah.... For some reason, when I tried first, it didn't work, not it does.